### PR TITLE
Models

### PR DESCRIPTION
**Pull request by frankcdlc**
models to:
Types
Requirements
Professional

### CODE REVIEW

File: app/models/type.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Type < ApplicationRecord
+  # validations
+  validates :name, uniqueness: true
+  validates :name, presence: true
+  validates :name,. inclusion: {in: %w("full-time part-time),,
+        message: "%{value} is not a valid type"}
```
Comment (Mhatw) [inline app/models/type.rb:6]: looks great, can you check typo and add more types like "freelance, remote"

**Review by alexisparave94 — CHANGES_REQUESTED** [review]
I guess you confuse with column enmail with column name of companies table

File: app/models/type.rb (hunk 1 above)
Comment (frankcdlc) [reply app/models/type.rb]: Thanks for the recomendations Mhatw, the changes were made

File: db/migrate/20220712220400_add_index_to_company.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class AddIndexToCompany < ActiveRecord::Migration[7.0]
+  def change
+    add_column :companies, :email, :string
+    add_index :companies, :email, unique: true
+  end
```
Comment (alexisparave94) [inline db/migrate/20220712220400_add_index_to_company.rb:5]: I guess you confuse the name of the column email, it should be name

Comment (frankcdlc) [reply db/migrate/20220712220400_add_index_to_company.rb]: You are right, thanks for your corrections

**Pull request merged**